src/test/resources/features/refapp-2.x/06-user/users.feature

```diff
@@ -1 +1,31 @@
-Feature: Users Management
\ No newline at end of file
+Feature: User Account Management
+
+  Background:
+    Given a user clicks on system administartion app from home page
+    Then the system loads system administration page
+    When a user clicks on manage accounts app
+    Then the system loads manage acccount page
+
+  @selenium
+  @userAccount
+  Scenario: Create user accounts
+    # User story: Creating user account for Data clerk
+    When a user clicks add new account button
```

Comment:
We need to make use of cucumber logical functions, I can see where you are using @when annotation for three consecutive times. Have you read through how we can re-use the same feature file in multiple scenarios. This will reduce code readability and quality[, checkout the very nice approach here ](https://johnfergusonsmart.com/1136-2/)

Reply:
@sherrif10 Thanks for the resource and after have exhausted it after reading  through the doc!

 Here is one things I would love to learn:

1. Do we only use a Gherkin syntax once in a scenario eg `When` should appear once in a scenario ?

Reply:
> 1. Do we only use a Gherkin syntax once in a scenario eg `When` should appear once in a scenario ?

This sounds so excellent, I wanted to also comment on that, so I think you can as well improve that.